lib/cocoapods-core/specification.rb

```diff
@@ -34,18 +34,18 @@ class Specification
     # @param  [String] name
     #         the name of the specification.
     #
-    # @param [Bool] test_specification
-    #        Whether the specification is a test specification
+    # @param [Symbol] type
+    #        the type of specification.
     #
-    def initialize(parent = nil, name = nil, test_specification = false)
+    def initialize(parent = nil, name = nil, type = :root)
```

Comment:
This is probably a breaking change for anything that used `Pod::Specification.new` ?